File: source/dotnet/Library/AdaptiveCards/Rendering/AdaptiveHostConfigParseResult.cs
namespace AdaptiveCards.Rendering
{
    public class AdaptiveHostConfigParseResult
    {
        public AdaptiveHostConfig HostConfig { get; }

        internal AdaptiveHostConfigParseResult(AdaptiveHostConfig hostConfig)
        {
            HostConfig = hostConfig;
        }
    }
}

File: source/winappsdk/Renderer/lib/AdaptiveHostConfigParseResult.cpp
// Copyright (c) Microsoft Corporation. All rights reserved.
// Licensed under the MIT License.
#include "pch.h"

#include "AdaptiveHostConfigParseResult.h"
#include "AdaptiveHostConfigParseResult.g.cpp"

namespace winrt::AdaptiveCards::Rendering::WinUI3::implementation
{
    AdaptiveHostConfigParseResult::AdaptiveHostConfigParseResult(WinUI3::AdaptiveHostConfig const& value) :
        Errors{winrt::single_threaded_vector<ObjectModel::WinUI3::AdaptiveError>()}
    {
        HostConfig = value;
    }
}
